web/.flowconfig

```diff
@@ -27,6 +29,10 @@
 module.system.node.resolve_dirname=node_modules
 esproposal.optional_chaining=enable
 
+; TODO Remove and fix flow errors to upgrade to v0.143.0
+; https://github.com/facebook/flow/releases/tag/v0.143.0
+types_first=false
```

Comment:
We have to solve this when we want to upgrade to current version

Reply:
I would say we switch to typscript before doing this flow upgrade :D

Reply:
Don't we also have to use a specific flow version with each react version?

Reply:
yes, but this change is for web, right?

Reply:
Ahh I see, so it is only important for react-native to stick to the same version?